### ISSUE TITLE
Make status icon optional

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#98]
```
The status icon as used by Pino violates the Gnome HIG. Could you make it 
optional (and exit on close)?
Thanks for your efforts!
```

Original issue reported on code.google.com by `bigmike.MK` on 18 Feb 2010 at 5:16

**Comment by GoogleCodeExporter** [comment]
```
You don't want to see any icon in a tray or you mean just the current logo?
```

Original comment by `tro...@gmail.com` on 18 Feb 2010 at 5:24

**Comment by GoogleCodeExporter** [comment]
```
In my case I do not use tray and would be very useful to be able to disable in 
pino.
```

Original comment by `manuel....@gmail.com` on 20 Feb 2010 at 4:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 20 Feb 2010 at 10:40
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I didn't make this very clear before. I mean, there should not be any 
icon in the 
notification area (the "tray") by default. However there MAY be an icon when 
unread messages are 
available. The program shouldn't "hide" in the tray, when the window is closed, 
it should exit 
then.

The Gnome Human Interface Guidelines say:
* Only core GNOME programs may perpetually display an icon in the status area.
* Non-core programs for which a perpetual icon may be useful must default to 
not perpetually 
showing the icon. Users may select to enable a perpetual icon for the 
application as a 
preference.

http://library.gnome.org/devel/hig-book/stable/desktop-notification-area.html.de

```

Original comment by `bigmike.MK` on 20 Feb 2010 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
Closed by e8a91f7a0518 revision.
```

Original comment by `tro...@gmail.com` on 20 Feb 2010 at 4:22
- Changed state: **Fixed**
